### PR TITLE
add ended_by to calls

### PR DESCRIPTION
**Pull request by ruslandoga**

### CODE REVIEW

File: lib/t/calls.ex [hunk 1]
```diff
@@ -147,12 +147,14 @@ defmodule T.Calls do
...
   end
 
-  # TODO remove the ones hung up by user_id
   @spec missed_calls_q(Ecto.UUID.t(), Keyword.t()) :: Ecto.Query.t()
   defp missed_calls_q(user_id, opts) do
     Call
     |> where(called_id: ^user_id)
+    # call hasn't been picked up
     |> where([c], is_nil(c.accepted_at))
+    # and call hasn't been declined
```
Comment (akhrail1996) [inline lib/t/calls.ex:156]: declined call is not exactly "missed", but still needs to be displayed for ability to conveniently call back

Comment (ruslandoga) [reply lib/t/calls.ex:156]: @akhrail1996 if a call has been declined, it should be stored in ios already though, right?

Comment (ruslandoga) [reply lib/t/calls.ex:156]: `missed calls` are supposed to only be the ones ios might not know about.

Comment (ruslandoga) [reply lib/t/calls.ex:156]: If however you consider users having multiple devices, we'd need to return all calls, even successful ones, on feed channel join.

Comment (akhrail1996) [reply lib/t/calls.ex:156]: declined calls are not stored in local db yet, though they are inserted in feed
therefore on app refresh they will be lost, which is bad

yeah, probably returning all calls will work best

Comment (ruslandoga) [reply lib/t/calls.ex:156]: https://github.com/getsince/test3/issues/298

**Pull request merged**